tests/spark_autologging/test_spark_datasource_autologging.py

```diff
@@ -106,6 +107,7 @@ def failing_req_mock(*args, **kwargs):
             time.sleep(1)
 
 
+@pytest.mark.skip(reason='fail on spark 3')
```

Comment:
Looks like almost all the tests fail. Can we fix the datasource autologging before this PR?